Makefile

```diff
@@ -1389,10 +1388,15 @@ lint-js-fix:
 
 .PHONY: lint-js
 .PHONY: lint-js-doc
+.PHONY: lint-js-build
+
+lint-js-build:
+	cd tools/eslint && $(call available-node,$(run-npm-ci))
```

Comment:
Will this change be executed whenever someone runs `make lint-js` or `make lint-js-fix`?

Reply:
Yes, `lint-js-build` is a dependency of both `lint-js`, `lint-js-doc`, and `lint-js-fix`

Reply:
This is unnecessary. We should cache the result and should not trigger npm ci everytime linter is run except eslint is updated.

Reply:
Now it will cache the data via a lintstamp.